package.json

```diff
@@ -90,6 +90,7 @@
     "native-watchdog": "^1.4.1",
     "node-pty": "1.1.0-beta5",
     "tas-client-umd": "0.1.8",
+    "util": "^0.12.4",
```

Comment:
We can probably kill util too, it's used in exactly on place in the browser unit tests to promisify `tree-kill`

Reply:
I believe it's used for webpack to compile extensions for browser targets. @aeschli please confirm.